ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/Milestone.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.spec.logic;
+
+public enum Milestone {
```

Comment:
`Milestone` works for me - or maybe `SpecMilestone`?  But I'd probably move this to `tech.pegasys.teku.spec.constants` or even just `tech.pegasys.teku.spec`.

Reply:
I like `SpecMilestone` and have gone with putting it in the `tech.pegasys.teku.spec` package as it seems like a top level kind of concept.